### PR TITLE
feat: add conda envrionment setup file

### PR DESCRIPTION
**Pull request by mprorock**
Adds a conda env file for those of us that live in conda land - also helps pin versions of packages

### CODE REVIEW

**Review by SteveLasker — APPROVED** [review]
Great add, @mprorock 
LGTM

File: scitt-api-emulator.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  - wheel=0.38.4
+  - pip:
+    - asn1crypto==1.5.1
+    - cbor2==5.4.6
+    - certvalidator==0.11.1
+    - cose==0.9.dev8
+    - ecdsa==0.18.0
+    - oscrypto==1.3.0
+    - six==1.16.0
+prefix: /home/mprorock/conda/envs/scitt
```
Comment (letmaik) [inline scitt-api-emulator.yml]: I guess this isn't supposed to be here?

**Review by letmaik — CHANGES_REQUESTED** [review]
Looks good, thanks!

File: scitt-api-emulator.yml (hunk 1 above)
Comment (mprorock) [reply scitt-api-emulator.yml]: hence eyes :)

Comment (mprorock) [inline scitt-api-emulator.yml]:
```suggestion
```

Comment (mprorock) [reply scitt-api-emulator.yml]: that should be removed now

**Comment by mprorock** [comment]
@letmaik / @SteveLasker what merge policy are we looking for here - assume rebase and merge? 

policy wise - after 2 approvals and X time - something we probably need to jot down, and I am happy to do so in the readme, or up a level at scitt-community governance and point to that policy from readme here

**Comment by letmaik** [comment]
@mprorock Maybe a separate CI job that tests using conda would make sense here. Here's a good example: https://github.com/marketplace/actions/setup-miniconda#use-a-default-shell

**Comment by mprorock** [comment]
> @mprorock Maybe a separate CI job that tests using conda would make sense here. Here's a good example: https://github.com/marketplace/actions/setup-miniconda#use-a-default-shell

definitely agree - do we want that as part of this PR, or as a new one so separate config/docs from CI actions?

**Comment by letmaik** [comment]
Separate PR is fine. Let's merge this one first.

**Review by letmaik — APPROVED** [review]

**Pull request merged**